feg/radius/src/modules/coadynamic/coa_dynamic_test.go

```diff
@@ -70,6 +68,16 @@ func TestCoaDynamic(t *testing.T) {
 	require.Equal(t, res.Code, radius.CodeDisconnectACK)
 }
 
+func waitForRadiusServerToBeReady(secret []byte, port int) {
+	MaxRetries := 10
```

Comment:
Should we return an error in this function? How would we be able to tell if there was an issue with `radius.Exchange`. Previously, we handled possible errors in `TestCoaDynamic`.

Reply:
Good point, I made it return an error if it exceeds the number of retries and assert that the returned error is nil.

Reply:
The point of radius.exchange is that it returns an error as long as the radius server is not yet running, and the function just retries the exchanges a couple of times until the server is up and responsive.